service/src/main/kotlin/fi/espoo/evaka/reports/AttendanceReservationReportController.kt

```diff
@@ -481,7 +481,7 @@ children AS (
   LEFT JOIN daycare_group g ON g.daycare_id = u.id AND g.id = coalesce(bc.group_id, dgp.daycare_group_id)
   WHERE u.id = ${bind(unitId)}
     AND (${bind(groupIds)}::uuid[] IS NULL OR g.id = ANY(${bind(groupIds)}))
-    AND extract(isodow FROM date) = ANY(u.operation_days)
+    AND extract(isodow FROM date) = ANY(coalesce(u.shift_care_operation_days, u.operation_days))
```

Comment:
Should we check service_need and shift_care here?

Reply:
This is probably correct since this is only used for filtering the rows and this anyway doesn't exclude e.g. holidays. I'll see if there's feedback from PO testing